app/routes/surveys/components/SubmissionsPage.js

```diff
@@ -0,0 +1,66 @@
+// @flow
+
+import * as React from 'react';
+import styles from './surveys.css';
+import type { SubmissionEntity } from 'app/reducers/surveySubmissions';
+import type { SurveyEntity } from 'app/reducers/surveys';
+import { DetailNavigation } from '../utils.js';
+import { Content, ContentSection, ContentMain } from 'app/components/Content';
+import LoadingIndicator from 'app/components/LoadingIndicator';
+import { Link } from 'react-router';
+import AdminSideBar from './AdminSideBar';
+import type { ActionGrant } from 'app/models';
+
+type Props = {
+  submissions: Array<SubmissionEntity>,
+  addSubmission: SubmissionEntity => Promise<*>,
+  deleteSurvey: number => Promise<*>,
+  survey: SurveyEntity,
+  children: React.Element<*>,
+  actionGrant: ActionGrant,
+  isSummary: boolean
+};
+
+const SubmissionPage = (props: Props) => {
```

Comment:
The file is called `SubmissionsPage.js`, and this is called `SubmissionPage` without an `s`.. Typo?

Reply:
Heh yeah, good point. I'll choose one and stick to it.

Reply:
Aaand, there is another `SubmissionPage`